src/components/presentational/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,41 @@
+import "./ProgressBar.css";
+import React, { ReactElement, useEffect } from "react";
+
+export interface ProgressBarProps {
+	background?: string;
+	fill?: string;
+	fillPercent: number;
+	steps?: [
+		{
+			stepPercent: number;
```

Comment:
"step" prefix here seems unnecessary since this will be in a "steps" array; so would just go with percent/icon